pandas/io/gbq.py

```diff
@@ -148,6 +151,43 @@ def get_credentials(self):
 
         return credentials
 
+    def get_credentials_for_service_account(self):
+        from oauth2client.client import SignedJwtAssertionCredentials
+
+        try:
+            with open(self.private_key_json_path) as f:
```

Comment:
you can be more helpful here as this could also be a string, or is this always how this is done? (always a file)